yarn.lock

```diff
@@ -1581,12 +1581,21 @@ acorn-jsx@^5.3.1:
   resolved "https://registry.yarnpkg.com/acorn-jsx/-/acorn-jsx-5.3.2.tgz#7ed5bb55908b3b2f1bc55c6af1653bada7f07937"
   integrity sha512-rq9s+JNhf0IChjtDXxllJ7g41oZk5SlXtp0LHwyA5cejwn7vKmKp4pPri6YEePv2PU65sAsegbXtIinmDFDXgQ==
 
+acorn-node@^1.3.0:
```

Comment:
Please run [`yarn-deduplicate`](https://www.npmjs.com/package/yarn-deduplicate) to minimize the number of dependencies we have to install.

Reply:
Done in 7d94d48.